### PR TITLE
Validation stream: provide validator's master key (FIXES: #3005)

### PR DESCRIPTION
**Pull request by undertome**
* Provide proposing validator's master key in the validation stream
  subscription JSON responses.

### CODE REVIEW

**Comment by undertome** [comment]
FYI this pull request doens't introduce breaking changes. The current development head (38f82115f7b53c10464a16fa27df7825140d961f) also causes the Travis build to fail in the same spot. This may be due to Xcode 11 (11A420a) being released on 9/20/19.

File: src/ripple/app/misc/NetworkOPs.cpp [hunk 1]
```diff
@@ -1721,16 +1721,27 @@ void NetworkOPsImp::pubValidation (STValidation::ref val)
     {
         Json::Value jvObj (Json::objectValue);
 
+        auto signerPublic = val->getSignerPublic();
```
[the remark this answers is not in the document]
Comment (undertome) [reply src/ripple/app/misc/NetworkOPs.cpp]: Done

File: src/ripple/app/misc/NetworkOPs.cpp [hunk 2]
```diff
@@ -1721,16 +1721,27 @@ void NetworkOPsImp::pubValidation (STValidation::ref val)
...
         jvObj [jss::full]                  = val->isFull();
         jvObj [jss::flags]                 = val->getFlags();
         jvObj [jss::signing_time]          = *(*val)[~sfSigningTime];
 
+        // TODO: Should we prefer this approach versus
+        //       'app_.validators().getTrustedKey()'?
+        //       Requires information about the definition
+        //       of a 'trusted' key.
+        auto masterKey = app_.validatorManifests().getMasterKey(signerPublic);
+        jvObj [jss::master_key] = toBase58(
```
[the remark this answers is not in the document]
Comment (undertome) [reply src/ripple/app/misc/NetworkOPs.cpp]: Done

**Comment by undertome** [comment]
>  LGTM
> 
> One nit: I'd change the git commit message to be more consistent with the other commit messages already in rippled

I was copying Mike Ellery's commit message style. Are one liners preferred?

**Comment by seelabs** [comment]
@undertome The body of the commit message is fine, I'm flagging the summary line. If I do `git log --oneline` I see `FIXES: #3005`. We usually have a summary line that gives a brief indication of what's fixed without going back to the issue. It's good to also include the issue in parens.

**Comment by undertome** [comment]
> @undertome The body of the commit message is fine, I'm flagging the summary line. If I do `git log --oneline` I see `FIXES: #3005`. We usually have a summary line that gives a brief indication of what's fixed without going back to the issue. It's good to also include the issue in parens.

Done

**Comment by undertome** [comment]
**pre-emptive self nit**: for branches I create in the future, is there a preferred branch naming syntax or no?

**Pull request closed without merging**